ote/src/cljs/ote/views/admin/interfaces.cljs

```diff
@@ -119,7 +119,8 @@
 
 (defn interface-list [e! app]
   (let [{:keys [loading? results filters]}
-        (get-in app [:admin :interface-list])]
+        (get-in app [:admin :interface-list])
+        fd (js/Date. 0)]
```

Comment:
what is fd?

Reply:
it's meant to be short for "first date", probably not a good symbol name though.